mavros/src/plugins/home_position.cpp

```diff
@@ -0,0 +1,104 @@
+/**
+ * @brief HomePosition plugin 
+ * @file home_position.cpp
+ * @author Thomas Stastny <thomas.stastny@mavt.ethz.ch>
+ *
+ * @Plugin to publish home position mavlink messages in ROS
+ * @addtogroup plugin
+ * @{
+ */
+/*
+ * Copyright 2017 Thomas Stastny.
+ *
+ * This file is part of the mavros package and subject to the license terms
+ * in the top-level LICENSE file of the mavros repository.
+ * https://github.com/mavlink/mavros/tree/master/LICENSE.md
+ */
+
+#include <list>
+
+#include <mavros/mavros_plugin.h>
+#include <pluginlib/class_list_macros.h>
+
+#include <mavros_msgs/HomePosition.h>
+
+namespace mavplugin {
```

Comment:
You are not applying new plugin API. Please check how currently the plugins are implemented and update please